About Me.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sarah "Nicky" Rhymes-portfolio,resume and about me </title>
+</head>
+<body>
+  <header>
+      <nav>
+        <img src= "Images/My Logo.jpg" alt="Nicky Rhymes logo">
+          <ul>
+            <li><a href="https://www.facebook.com/sarah.rhymes.90/">Facebook</a></li>
+            <li><a href="https://twitter.com/Srhymes345" >Twiter</a></li>
+            <li><a href="https://www.pinterest.com/IceWitch1010/_saved/">Pintrest</a></li>
+            <li><a href="https://www.instagram.com/the_shadow_witch1010/"> Instagram</a></li>
+             </ul>           
+          <ul>
+            <li><a href="http://127.0.0.1:5500/Protfolio.html">Protolio</a></li>
+            <li><a href="http://127.0.0.1:5500/Digital%20Resume.html"> Digital Resume</a></li>
```

Comment:
we also want to avoid using filenames with spaces as it creates issues. instead of `Digital Resume.html` rename to `digital-resume.html` same for `about-me.html` or you can simplify to `resume.html` or `about.html`